src/views/community.html

```
@@ -0,0 +1,20 @@
+{% set pageName = 'Community' %}
+
+{% block main %}
+<div class="govuk-width-container">
+  <h1 class="govuk-heading-xl"></h1>
+  <h1 class="govuk-heading-xl">Community</h1>
+  <span class="govuk-body">You can</span>
+  <a href="https://opendigitalplanning.org/about">
+  <span class="govuk-body">get involved with community events and discussions </span>
+  </a>
+  <span class="govuk-body">to help improve this service.</span>
+  </a>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </a> ]

(tag-pair)